onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -103,7 +105,9 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
                   (global_context_.OpenVINO_Version.at(0) >= 2024 && global_context_.OpenVINO_Version.at(1) > 2))) {
         // Optimized OV compile_model API is supported with AUTO from version 2024.3 and above
         // Inputs with static dimenstions
-        const std::string model = model_proto.SerializeAsString();
+        const std::string model = model_proto->SerializeAsString();
```

Comment:
Model Proto changes look ok but should we keep string as constant as we want to delete it later ?